packages/client-hac/src/index.ts

```
@@ -761,6 +841,101 @@
+                const response = await generateMessageResponse({
+                    runtime: runtime,
+                    context,
+                    modelClass: ModelClass.LARGE,
+                });
+
+                if (!response) {
+                    res.json({ pass: false });
+                    return;
+                }
+                let pass = "error";
+                if (response && response.decision) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)